src/app.js

```diff
@@ -215,48 +217,134 @@ export default function (app) {
       var realValue = element[name]
 
       if (name === "onUpdate") {
-        defer(value, element)
+          defer(value, element)
 
       } else if (value !== oldValue || realValue !== value) {
         setElementData(element, name, value, oldValue)
       }
     }
   }
 
-  function patch(parent, element, oldNode, node) {
-    if (oldNode === undefined) {
-      element = parent.appendChild(createElementFrom(node))
-
-    } else if (node === undefined) {
+  function batchRemove (parent, element, node) {
       batch.push(parent.removeChild.bind(parent, element))
-
-      if (oldNode && oldNode.data && oldNode.data.onRemove) {
-        defer(oldNode.data.onRemove, element)
+      if (node.data && node.data.onRemove) {
+        defer(node.data.onRemove, element)
       }
+  }
+
+  function patch (parent, element, oldNode, newNode) {
+    if (oldNode === undefined) {
+      var n = createElementFrom(newNode)
+      parent.insertBefore(n, element)
+      element = n
 
     } else if (
```

Comment:
If you're referring to the "insert app directly into document.body" merge, then yes. This code has been updated specifically to adapt to that (to return the new element of a patch)